### PR TITLE
Feature/menu screen scroll

### PR DESCRIPTION
**Pull request by wbeltran777**
Yo yo, I did a quick implementation of the background scrolling. Let me know what you guys think!

### CODE REVIEW

**Comment by m4rtint** [comment]
Conflicts

**Comment by Gursaanj** [comment]
[Make the car background move](https://trello.com/c/vedAgHHs/193-make-the-car-background-move)

**Comment by Gursaanj** [comment]
Seems to loop through some empty background too!!, but other than that I think the speed is nice and smooth

Gotty broken screen slide

File: Assets/Script/UI/BackgroundBehaviour.cs [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+/// <summary>
+/// In charge of behaviour for scrolling background in title screen
+/// </summary>
+public class BackgroundBehaviour : MonoBehaviour
+{
+
+    [SerializeField] private float m_scrollSpeed = 2.0f;
+    private RectTransform m_backgroundCenter;
+    private Vector3 m_originalPosition;
+    private Image m_backgroundImage;
```
Comment (Gursaanj) [inline Assets/Script/UI/BackgroundBehaviour.cs]: seems like m_backgroundImage is not being used

Comment (wbeltran777) [reply Assets/Script/UI/BackgroundBehaviour.cs]: Deleted it, also lol I promise I ran this before submitting. I'll look into why the image isn't scrolling correctly.

Comment (m4rtint) [reply Assets/Script/UI/BackgroundBehaviour.cs]: check up on different resolutions.

Comment (m4rtint) [reply Assets/Script/UI/BackgroundBehaviour.cs]: U might need to get the physical screen resolution, and then scroll according to the width.

**Comment by wbeltran777** [comment]
Fixed the scene conflicts, the new scrolling background is in a prefab called MainMenuBackground in the UI prefab folder. 
To try out the scrolling just drag the prefab to replace 'Image' in the Main Menu Scene.
Main Camera -> Canvas -> Image

**Comment by wbeltran777** [comment]
New prefab to drag in is Assets -> Prefab -> UI -> BackGroundScroll

Replace image in the main camera

Last implementation I used two images which did some weird stuff for distance when changing aspect ratios. 
This time it's just a long strip. 
The looping is a little bit janky, but I made the scrolling image long enough and the speed slow enough that it won't really be visible often. 
I know it's not an amazing fix, but I'd like to get something done before Saturday

**Pull request merged**